keyboards/varys/keyboard.json

```diff
@@ -0,0 +1,118 @@
+{
+    "manufacturer": "Vinay Komaravolu",
+    "keyboard_name": "Varys",
+    "maintainer": "Vinay Komaravolu",
+    "bootloader": "caterina",
```

Comment:
```suggestion
    "development_board": "promicro",
```

Reply:
This pcb is not using a promicro as this is a custom pcb with an embedded ATmega32u. I do not think it should be added.

Reply:
And microcontroller is using a caterina bootloader? I find that unlikely. It's more likely to be `atmel-dfu`.

Reply:
Hmmm I did some verification and doubled checked. It is indeed using the caterina bootloader for my ATMEGA32U4-MU MCU. Flashing through QMK MSYS also flashes for bootloader: caterina.
![image](https://github.com/user-attachments/assets/ff1d3bfb-6b63-46dc-85cd-be9005281141)